pages/ai-ecosystem/integrations.mdx

```diff
@@ -274,15 +275,24 @@ The [LangGraph](https://www.langchain.com/langgraph) framework enables users to
 build agentic applications. Memgraph now offers a toolkit for building agents
 that can autonomously interact with the Memgraph database. 
 
-Currently, the Memgraph toolkit supports the following tools:
+Currently, the Memgraph [toolkit](https://github.com/memgraph/ai-toolkit/blob/main/integrations/langchain-memgraph/langchain_memgraph/toolkits.py) supports the following tools:
+
+- [RunQueryTool]: Executes Cypher queries on the Memgraph database.
+- [RunShowIndexInfoTool]: Retrieves information about the indexes in the database.
+- [RunShowSchemaInfoTool]: Retrieves information about the schema in the database.
+- [RunShowConfigTool]: Retrieves information about the configuration of the database.
+- [RunShowStorageInfoTool]: Retrieves information about the storage engine of the database.
+- [RunShowTriggersTool]: Retrieves information about the triggers in the database.
+- [RunShowConstraintInfoTool]: Retrieves information about the constraints in the database.
+- [RunBetweennessCentralityTool]: Calculates the betweenness centrality of nodes in the graph.
+- [RunPageRankMemgraphTool]: Calculates the PageRank of nodes in the graph.
```

Comment:
Were these supposed to be links? or Code blocks? [ ]

Reply:
They were, but then I deleted the links and forgot about the `[]` will just bold the tools.